utils.nix

```diff
@@ -28,10 +28,12 @@ rec {
   node2nixProd = (
     (import (node2nixDrv false) { inherit pkgs nodejs; }).shell.override {
       dontNpmInstall = true;
+      buildInputs = [ nodePackages.node-gyp-build ];
     }
   ).nodeDependencies;
   node2nixDev = (import (node2nixDrv true) { inherit pkgs nodejs; }).package.overrideAttrs (attrs: {
     src = src;
+    buildInputs = attrs.buildInputs ++ [ nodePackages.node-gyp-build ];
```

Comment:
Have a check whether both require this override, or just one of `node2nixProd` and `node2nixDev`.

Reply:
Just having it in node2nixProd doesn't work while just having it in node2nixDev does.
Removing it from node2nixProd.